pkgs/development/python-modules/urllib3/default.nix

```diff
@@ -46,6 +46,10 @@ buildPythonPackage rec {
     trustme
   ];
 
+  disabledTests = [
+    "test_connection_closed_on_read_timeout_preload_false"
```

Comment:
fails only on aarch64. Disabling altogether will result in a mass rebuild.

Reply:
```
disabledTests = if stdenv.hostPlatform.isAarch64 then [  "test_connection_closed_on_read_timeout_preload_false" ] else null;
```

Reply:
pushed 2e0b261da8c4a08d642bcbf115ffbca5d69ff481 to staging-next

Reply:
I build urrlib3 on amd64 with --check and this was required to build the tests. I don't think we get around the mass rebuild.